Mage.Sets/src/mage/cards/c/CabalInquisitor.java

```diff
@@ -61,8 +61,8 @@ class ActivateAsSorceryConditionalActivatedAbility extends ActivatedAbilityImpl
 
     public ActivateAsSorceryConditionalActivatedAbility(Zone zone, Effect effect, ManaCosts cost, Condition condition) {
         super(zone, effect, cost);
-        this.condition = condition;
```

Comment:
Constructors must use direct assign as much as possible, not public/overrideable setters and getters:
https://stackoverflow.com/a/53291311/1276632

Reply:
The linked question says that the reasons getters and setters are a bad idea is because they can be overriden. So I've made them final to prevent overriding.

Reply:
Nope. Getters and setters in constructors is a bad. That's code style. If you want to catch and fix field usages then [use different access modifier in your refactor](https://stackoverflow.com/a/33627846/1276632).